src/components/ConfirmDeleteModal.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+export default function ConfirmDeleteModal({
```

Comment:
for consistency reasons for components it would be better to move the export default to the end of the function